components/molecule/accordion/src/settings.js

```diff
@@ -70,3 +70,8 @@ export const HEADER_ICON_POSITION = {
   LEFT: 'left',
   RIGHT: 'right'
 }
+
+export const HEADER_LABEL_WRAPS = {
+  NO_WRAP: 'noWrap',
+  WRAP: 'wrap'
+}
```

Comment:
🍆 unless you're thinking of adding more styles of wrapping in the future... wouldn't a `boolean` prop do the job? 🤔